apps/state/test/state/facility_test.exs

```diff
@@ -94,7 +94,7 @@ defmodule State.FacilityTest do
         |> State.Facility.filter_by()
         |> Enum.sort_by(& &1.id)
 
-      assert sorted_results == [by_id("3"), by_id("6"), by_id("701")]
+      assert sorted_results == [by_id("3")]
```

Comment:
Are there remaining tests about the sorting?

Reply:
There aren't any tests explicitly of sorting, but that seems like it would belong in the controller testing.

There are other tests which have multiple results and sort them by ID, but I interpreted those as sorting only so that we can be guaranteed equality when comparing lists.

Reply:
Ah, I see. I misunderstood the test. :+1: